ecosystem/sep-0010.md

```diff
@@ -23,7 +23,8 @@ The authentication flow is as follows:
 1. The client obtains a unique [`challenge`](#challenge), which is represented as specially formed Stellar transaction
 1. The client verifies that the transaction has an invalid sequence number 0.  This is extremely important to ensure the transaction isn't malicious.
 1. The client verifies that the transaction is signed by the `SIGNING_KEY` specified by the requested service's [SEP-1 stellar.toml](sep-0001.md).
-1. The client verifies that the transaction includes a Manage Data operation, and that the value contains the home domain used to request the challenge.
+1. The client verifies that the transaction has a single Manage Data operation with its source account set to the user's account and value set to a nonce value.
```

Comment:
Should we make it explicit that the client should **not** validate the contents of the `key` param? I want to avoid any confusion, since later on in the spec we give more strict instructions for the server-side implementation.

Reply:
We could make it more explicit, but I also think we are being sufficiently explicit that the home domain is no longer included in what should be verified. This returns the specification back to essentially what it said prior to this change. Let me know if you feel strongly about this, or can you make a suggestion for how to reword this?

Reply:
What do you think of this?
```suggestion
1. The client verifies that the transaction has a single Manage Data operation with its source account set to the user's account and value set to a nonce value. The client ignores the home domain included.
```

Reply:
Looks good!